docs/interfaces/desktopagent.md

```diff
@@ -113,10 +152,35 @@ ___
 
 ▸ **raiseIntent**(intent: *`String`*, context: *[Context](../#context)*, target?: *`String`*): `Promise`<[IntentResolution](intentresolution.md)>
 
-*Defined in [interface.ts:103](/src/interface.ts#L103)*
+*Defined in [interface.ts:191](/src/interface.ts#L191)*
 
 Raises an intent to the desktop agent to resolve.
 
+```javascript
+//raise an intent to start a chat with a given contact
+var intentR = await agent.resolve("StartChat",{version:"1.0.0",entities:[
```

Comment:
The result will be "action" including the application with the intents on it, not just the intent.